src/Test.UnitTests.Sarif.Driver/DriverUtilitiesTests.cs

```diff
@@ -4,12 +4,12 @@
 using System;
 using CommandLine;
 using FluentAssertions;
-using Microsoft.CodeAnalysis.Sarif.Multitool;
+using Microsoft.CodeAnalysis.Sarif.Driver;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool
+namespace Test.UnitTests.Sarif.Driver
```

Comment:
I don't agree w/this convention. Instead, all unit tests should have the same namespace as the core product types under test. In order to make this more convenient when creating test cases, we should modify every unit test project file to specify the product team default namespace.

To be clear, the namespace of this file s/be the same namespace where the type under test, DriverUtilities, exists. The rationale for this choice is that the test code will have, appropriately, implicit access to the same visible namespace as the product type. This eliminates the need for a using directive.

Reply:
Reverted. After my two current high-pri tasks I'm going to make a single pass and adjust every test file we have, and also modify `build.props` (if I can) to automatically take care of the default namespace for test projects.

---
In reply to: [315956789](https://github.com/microsoft/sarif-sdk/pull/1653#discussion_r315956789) [](ancestors = 315956789)